tests/integ/test_tools.py

```diff
@@ -284,22 +283,22 @@ def test_detr_segmentation_empty():
     assert result == []
 
 
-def test_clip():
-    img = ski.data.coins()
-    result = clip(
-        classes=["coins", "notes"],
-        image=img,
-    )
-    assert result["scores"] == [0.9999, 0.0001]
+# def test_clip():
+#     img = ski.data.coins()
+#     result = clip(
+#         classes=["coins", "notes"],
+#         image=img,
+#     )
+#     assert result["scores"] == [0.9999, 0.0001]
 
 
-def test_clip_empty():
-    result = clip(
-        classes=["coins", "notes"],
-        image=np.zeros((0, 0, 3)).astype(np.uint8),
-    )
-    assert result["scores"] == []
-    assert result["labels"] == []
+# def test_clip_empty():
+#     result = clip(
+#         classes=["coins", "notes"],
+#         image=np.zeros((0, 0, 3)).astype(np.uint8),
+#     )
+#     assert result["scores"] == []
+#     assert result["labels"] == []
 
```

Comment:
why is this commented?

Reply:
I did not removed because I also did not remove the clip function, just the references. What do you recommend: removing tests and function or removing only tests?

Reply:
you can still have the tests if you did not remove the function, right?

Reply:
I put the tests back